grails-app/conf/application-development.yml

```diff
@@ -0,0 +1,6 @@
+
+# Only add richly formatted logs when developing locally. Some terminals can't handle ANSI colors, so
+# we don't want to break any log processors in production. https://logback.qos.ch/manual/layouts.html
+logging:
+    pattern:
+        console: "%cyan(%date{ISO8601}) %highlight(%-5level) %magenta([%thread]) %yellow(%logger{40}): %message%n%xException"
```

Comment:
I originally tried doing this via Grails' environment properties:
```
environments:
    development:
        logging:
            pattern:
                console: <pattern with colors>
...
logging:
    pattern:
        console: <pattern without colors>
```
but unfortunately it didn't get picked up by Logback. I assume it's because the environments is a Grails thing.